project/cmake/addons/CMakeLists.txt

```diff
@@ -288,3 +294,7 @@ if(NEED_SUDO)
                     COMMAND sudo ${CMAKE_COMMAND} -E remove_directory ${ADDON_INSTALL_DIR}/
                     COMMAND sudo -k)
 endif()
+
+# add custom target "supported_addons" that returns all addons that are supported on this platform
+string(REPLACE ";" " " ALL_ADDONS_BUILDING "${ALL_ADDONS_BUILDING}")
+add_custom_target(supported_addons COMMAND ${CMAKE_COMMAND} -E echo "ALL_ADDONS_BUILDING: ${ALL_ADDONS_BUILDING}" VERBATIM)
```

Comment:
Is the `ALL_ADDONS_BUILDING: ` output prefix necessary? Would probably be easier to parse if it would just return  a list of addons. But I should get it to work with it as well.